go.mod

```diff
@@ -31,6 +31,7 @@ require (
 	github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0
 	github.com/hashicorp/golang-lru v0.5.3
 	github.com/hashicorp/hcl/v2 v2.0.0
+	github.com/jinzhu/copier v0.3.5 // indirect
```

Comment:
I would like to use the standard lib instead of adding a new one.
For example, we can marshal and re-unmarshal it to a new struct.

Reply:
> marshal and re-unmarshal it to a new struct

I re-implemented according to this method: 37f0be9e7da98c05bf12804a6c826651b21571cd